crates/flowctl/src/pagination.rs

```diff
@@ -120,27 +122,31 @@ where
         request: PaginationRequest,
     ) -> PageTurnerOutput<Self, PaginationRequest> {
         let resp: Vec<Item> = api_exec::<Vec<Item>>(request.builder.clone()).await?;
+        let response_length = resp.len();
 
         // Sometimes, it seems, we can get back more than the requested page size.
         // So far I've only seen this on a request of 1,000 and a response of 1,001.
-        if resp.len() >= request.page_size
+        if response_length >= request.page_size
```

Comment:
The comment above is explained by the range being inclusive, yeah? Might be good to remove or update it.

Reply:
Good call, removed!